### PR TITLE
added battery light monitor to cob4-2s bringup

### PR DESCRIPTION
**Pull request by benmaidel**

### CODE REVIEW

**Comment by fmessmer** [comment]
add for cob4-1, too!

requires https://github.com/ipa320/cob_command_tools/pull/130

**Comment by NadiaHG** [comment]
CR : ok

**Comment by fmessmer** [comment]
@ipa-bnm is still working on it...waiting for his "GO"

**Comment by benmaidel** [comment]
needs still some testing on robots

File: cob_bringup/robots/cob4-1.xml [hunk 1]
```diff
@@ -61,6 +61,10 @@
 		<include file="$(find cob_bringup)/tools/emergency_stop_monitor.launch" >
 			<arg name="robot" value="$(arg robot)" />
 		</include>
+		<include file="$(find cob_bringup)/tools/battery_light_monitor.launch" >
+			<arg name="robot" value="$(arg robot)" />
+			<arg name="light_service" value="/light_torso/set_light"/>
```
Comment (floweisshardt) [inline cob_bringup/robots/cob4-1.xml]: to be consistent with other monitors, we should move the config into a yaml file and not specify it in the launch file.

File: cob_bringup/tools/battery_light_monitor.launch [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```
Comment (floweisshardt) [inline cob_bringup/tools/battery_light_monitor.launch:1]: tabs vs spaces

File: cob_bringup/tools/battery_light_monitor.launch [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+<launch>
+
+	<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
+  <arg name="light_service" default="/light/set_light"/>
```
Comment (floweisshardt) [inline cob_bringup/tools/battery_light_monitor.launch]: see https://github.com/ipa320/cob_robots/pull/404/files#r52274981

File: cob_bringup/robots/cob4-1.xml (hunk 1 above)
Comment (benmaidel) [inline cob_bringup/robots/cob4-1.xml]: i think that remapping arguments should not stay inside a yaml file.

Comment (floweisshardt) [inline cob_bringup/robots/cob4-1.xml]: OK, let's keep that

**Comment by fmessmer** [comment]
FR ok

**Comment by NadiaHG** [comment]
FR: OK

**Pull request merged**